### ISSUE TITLE
Cannot change password SFTP

### ISSUE THREAD

**Issue by Pixel-Edwin** [Pterodactyl/Panel#566]
When I like to change a servers SFTP password the panel gives an error.
`An unknown error occured while attempting to update this server's SFTP settings. `

Laravel Logs:
```production.ERROR: GuzzleHttp\Exception\ServerException: Server error: `POST http://***:8080/server/password` resulted in a `500 Internal Server Error` response:
{
    "error": "Docker returned a non-zero exit code when attempting to execute a SFTP command. Code: 1"
}```
That `Code: 1` did I placed in to check what errorcode Docker returns.

Portforwaring seems ok. I can connect with SFTP using one of the other servers its credentials.
I already removed the SFTP docker container and image and restarted Docker and Wings many times.

Running Panel v0.6.4 and Daemon v0.4.3.

**Comment by Pixel-Edwin** [comment]
Update: I can change the SFTP password of newly created servers. Is there a manual way to set the passwords?

**Comment by Pixel-Edwin** [comment]
Fixed. The Daemon did not create a SFTP user for some of the servers. I created them manually using:
`docker exec -it <SFTPdockerID> scrappyuser -u <DockerContainerName> -p <SomeRandomPassword>`

After creating the user I was able to change the passwords in Pterodactyl Panel.

To prevent future issues: can the Daemon check if a SFTP user exists or not by changing the password? If not, then the Daemon have to create it.
